<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipsConfigurator</class>
 <widget class="QWidget" name="ShipsConfigurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="submarinesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Submarines</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondSubmarine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstSubmarine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="thirdSubmarine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fourthSubmarine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="destroyersLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Destroyers</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstDestroyer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondDestroyer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="thirdDestroyer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cruisersLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cruisers</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstCuiser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondCruiser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="battleshipLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battleship</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="battleshipEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createShipsButton">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>120</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create ships</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Random</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
